src/mumble/AudioInput.h

```diff
@@ -9,9 +9,11 @@
 #include <QElapsedTimer>
 #include <QtCore/QObject>
 #include <QtCore/QThread>
+#include <algorithm>
```

Comment:
The new includes should be in the cpp file, since that's where they are needed

Reply:
Done, not sure why I put those in the header in the first place.